flang/include/flang/Lower/FIRBuilder.h

```diff
@@ -343,10 +343,15 @@ mlir::Value readLowerBound(FirOpBuilder &, mlir::Location,
                            const fir::ExtendedValue &, unsigned dim,
                            mlir::Value defaultValue);
 
-/// Read extents from an BoxValue into \p result.
+/// Read extents from an BoxValue.
 llvm::SmallVector<mlir::Value> readExtents(FirOpBuilder &, mlir::Location,
                                            const fir::BoxValue &);
 
+/// Get extents from an ExtentdedValue. For fir::BoxValue and
+/// fir::MutableBoxValue, this will generate code to read the extents.
+llvm::SmallVector<mlir::Value> getExtents(FirOpBuilder &, mlir::Location,
+                                          const fir::ExtendedValue &);
```

Comment:
nit: We should actually be giving arguments names in declaration. I believe that is the prevailing style in MLIR/LLVM and it can improve understanding the code. (e.g., Here the name `result` was left as a dangling reference to something that was apparently elided at some point.)